src/Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs

```diff
@@ -13,6 +13,7 @@
 
 #endregion
 
+#if !UNIX
```

Comment:
Should we rename this to UnblockFile.Windows.cs? 

---
In reply to: [133130873](https://github.com/PowerShell/PowerShell/pull/4567#discussion_r133130873) [](ancestors = 133130873)